fe/package.json

```diff
@@ -1,8 +1,14 @@
 {
   "dependencies": {
+    "@apollo/react-hooks": "^3.1.5",
     "@types/uuid": "^8.0.0",
     "@types/yup": "^0.29.2",
+    "apollo-boost": "^0.4.9",
```

Comment:
I'm keen on trying out the [Apollo Client 3 RC](https://github.com/apollographql/apollo-client/blob/v3.0.0-rc.2/CHANGELOG.md) to see if it resolves out caching woes, but that can be for another day 🙂